Automattic/jetpack
# Add Send Email Preview feature

Part of https://github.com/Automattic/wp-calypso/issues/76134

## Proposed changes:

* Adds a "Preview" button to the Pre Publish Newsletter Panel.
* The "Preview" button opens a "Send a test email" modal showing the current user email and a "Send" button.

<img width="1200" alt="Screenshot 2023-06-20 at 15 46 49" src="https://github.com/Automattic/jetpack/assets/104869/f6e434db-941c-424b-812d-f375007c4797">

Future iteration will use https://github.com/WordPress/gutenberg/pull/50605

### Other information:


## Jetpack product discussion
peKye1-6z-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

- Build/Run this branch on your test site.
- Create a new post on your self-hosted site.
- Click Publish, in Pre Publish, click "Preview" at the bottom of the Newsletter panel.
- A "Send a test email" modal should open.
- Click on the "Send" button.
- You should receive a test email!
